apply_dimension.json

```
@@ -48,16 +56,13 @@
         }
     },
     "returns": {
-        "description": "A data cube with the newly computed values for the specified . The resolution, cardinality and the number of dimensions are the same as for the original data cube.",
+        "description": "A data cube with the newly computed values for the specified. The resolution, cardinality and the number of dimensions are the same as for the original data cube.",
```

Cardinality may change

```suggestion
        "description": "A data cube with the newly computed values for the specified. The resolution and the number of dimensions are the same as for the original data cube.",
```